File: app/src/main/java/com/rpifinal/hitema/services/CloudMessagingService.java
package com.rpifinal.hitema.services;

import android.util.Log;
import com.google.firebase.messaging.FirebaseMessagingService;
import api.UserHelper;

public class CloudMessagingService extends FirebaseMessagingService {

    @Override
    public void onNewToken(String s) {

        super.onNewToken(s);
    }

    private void sendRegistrationToServer(String uid, String token) {

        UserHelper.updateToken(uid, token).addOnSuccessListener(aVoid ->
                Log.i("CMService", "Token updated"));
    }
}

File: app/src/main/java/oldFiles/services/CloudMessagingService.kt
package oldFiles.services

import android.util.Log
//import oldFiles.api.UserHelper.updateToken
import com.google.firebase.messaging.FirebaseMessagingService

class CloudMessagingService : FirebaseMessagingService() {
    override fun onNewToken(s: String) {
        super.onNewToken(s)
    }

/*    private fun sendRegistrationToServer(uid: String, token: String) {
        updateToken(uid, token).addOnSuccessListener { aVoid: Void? -> Log.i("CMService", "Token updated") }
    }*/
}